lib/elixir/lib/file.ex

```diff
@@ -10,6 +10,11 @@ defmodule File do
   via `cp/3` and remove files and directories recursively
   via `rm_rf/1`.
 
+  Paths given to functions in this module can be either relative
+  paths or aboslute paths. Shell conventions like `~` are
+  not expanded automatically. To use paths like `~/Downloads`,
+  you can use `Path.expand/1-2` to expand your path to an absolute path.
```

Comment:
Should we mention that relative paths are relative to the current directory, that is, `File.cwd!/0`?